api-reference/_hidden/CellAddress/CellAddress.md

```diff
@@ -4,7 +4,8 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A cell used as a start position for export.
```

Comment:
This should be reverted because CellAddress can be used not only by the export API.